src/Modules/CalcPerform.lua

```diff
@@ -1005,7 +1005,7 @@ function calcs.perform(env, skipEHP)
 				env.minion.modDB:AddList(env.player.itemList["Weapon 1"].slotModList[1])
 			end
 			if env.player.itemList["Weapon 2"] and env.player.itemList["Weapon 2"].type == "Quiver" then
-				env.minion.modDB:AddList(env.player.itemList["Weapon 2"].modList)
+				 env.minion.modDB:ScaleAddList(env.player.itemList["Weapon 2"].modList, env.widowHailModifier or 1)
```

Comment:
There appears to be an extra space after the indentation tabs?

Reply:
ty, fixed